lib/preview.bash

```diff
@@ -1,19 +1,20 @@
-if [[ "${BASH_PREVIEW:-}" ]];
-then
-  unset BASH_PREVIEW #Prevent infinite looping
-  echo "
+# shellcheck shell=bash
+#
+# Displays the prompt from each _Bash It_ theme.
 
-  Previewing Bash-it Themes
+function _bash-it-preview() {
```

Comment:
maybe we should provide a way to do this from the command line? or at least doc about it?

Reply:
Absolutely! In fact, that's the motivation for the change. But it's extremely hard to make non-conflicting changes to `bash-it`, so I left that for a subsequent PR after all these other changes are eventually merged (including your `profile` subcommand).

Reply:
lets try to have this as part of this PR!

Reply:
Ok, I've added it to this PR and even updated the documentation.